test/run

```diff
@@ -1067,12 +1039,6 @@ testCIEnvVars() {
   assertCapturedSuccess
 }
 
-# If compile fails, test-compile should also fail
-testCICompileFails() {
-  testCompile "failing-build"
-  assertCapturedError
-}
-
```

Comment:
asserting error is captured is already checked for when "failing-build" is compiled elsewhere